src/pages/_document.tsx

```diff
@@ -4,7 +4,7 @@ export default function Document() {
   return (
     <Html lang='en'>
       <Head>
-        <link rel='icon' href='/favicon.ico' />
+        <link rel='icon' href='/dflogo.svg' />
```

Comment:
I asked people on Twitch which one they like more, and they said they like the logoDF one.

The first one without the text is really nice too though - I might ask them again tomorrow with these two options.